content/docs/1.6.0/references/settings.md

```diff
@@ -101,6 +102,12 @@ Define the policy to use when a node with the last healthy replica of a volume i
 
 Longhorn will generate system snapshot during replica rebuild, and if a user doesn't setup a recurring snapshot schedule, all the system generated snapshots would be left in the replica, and user has to delete them manually, this setting allow Longhorn to automatically cleanup system generated snapshot before and after replica rebuild.
 
+#### Automatically Clean up Outdated Snapshots of Recurring Backup Jobs
+
+> Default: `true`
+
+When running a recurring backup job, Longhorn takes a new snapshot before creating the backup. Longhorn retains only the snapshot used by the last backup job even if the value of the retain parameter is not 1. This setting ensures that the retained snapshots directly correspond to the backups on the remote backup target.
```

Comment:
The description seems slightly incorrect. I suggest changing it as below, and this can make it consistent with other boolean settings. 

```suggestion
If enabled, when running a recurring backup job, Longhorn takes a new snapshot before creating the backup. Longhorn retains only the snapshot used by the last backup job even if the value of the retain parameter is not 1. 

If disabled, this setting ensures that the retained snapshots directly correspond to the backups on the remote backup target.
```